frontend/src/pages/choose-role/choose-role.js

```diff
@@ -0,0 +1,70 @@
+import { ENDPOINTS } from '../../api/config';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Form from '../../components/input-form/input-form';
+import Button from '../../components/button/button';
+import ToggleButtonGroup from '../../components/toggle-button-group/toggle-button-group';
+import './choose-role.css';
+
+
+const ChooseRole = () => {
+
+    const [selectedRole, setSelectedRole] = useState(1); // Default to "Startup" (1)
```

Comment:
Consider initializing the role state dynamically if the current user has a saved role. This improves user experience by showing the default selection based on their existing role.
```
const [selectedRole, setSelectedRole] = useState(() => {
    const savedRole = localStorage.getItem('selectedRole');
    return savedRole ? parseInt(savedRole, 10) : 1; // Default to "Startup"
});
```